jobs/job_utils.py

```diff
@@ -83,7 +86,9 @@ def get_model_class(kind):
     Raises:
         KindError. Internally raised by _lookup_model when the kind is invalid.
     """
-    return datastore_services.Model._lookup_model(kind)  # pylint: disable=protected-access
+    return cast(
+        Type[datastore_services.Model],
+        datastore_services.Model._lookup_model(kind))  # type: ignore[attr-defined]  # pylint: disable=protected-access
```

Comment:
Why does it need to be cast here?

Reply:
The casts are needed because MyPy doesn't recognize any of the private methods, I suspect we'll be able to remove them once me move to using `requirements.txt` though.

Reply:
Wait, you just need to add the definition to the stubs folder (stubs/google/cloud/ndb/__init__.py), right?.

Reply:
Done.